tests/test_airdrops.py

```diff
@@ -29,30 +30,20 @@ def test_airdrops_metadata():
             assert os.path.exists(airdrop['icon_path'])
 
 
-def test_csvs_and_jsons():
-    with open('airdrops/index_v2.json', 'r') as f:
+def test_files_and_jsons():
+    with open('airdrops/index_v3.json', 'r') as f:
         airdrop_index = json.load(f)
 
     for airdrop in airdrop_index['airdrops'].values():
         # verify the csv_hash is correct
-        if 'csv_hash' in airdrop:
-            with open(airdrop['csv_path'], 'br') as f:
-                csv_b = f.read()
-
-            assert airdrop['csv_hash'] == hashlib.sha256(csv_b).hexdigest(), f'Invalid hash for {airdrop["csv_path"]}'
-            csv_rows_b = csv_b.split(b'\n')
-            assert len(csv_rows_b) > 0, f'{airdrop["csv_path"]} should have LF EoL sequence with multiple rows'
-            assert csv_rows_b.pop(0).startswith(b'address,amount'), f'{airdrop["csv_path"]} airdrop missing `address,amount` header'
-
-            for i, row_b in enumerate(csv_rows_b):
-                row = row_b.decode('ascii')
-                # test that all addresses are checksummed
-                address = row.split(',')[0]
-                assert is_checksum_address(address), f'{airdrop["csv_path"]} address {address} is not checksummed at row {i+2}'
-
-                # check that the CSV have LF EoL sequence
-                assert not row.endswith('\r'), f'{row_b} in {airdrop["csv_path"]} should have LF EoL sequence at row {i+2}'
+        if 'file_hash' in airdrop:
+            with open(airdrop['file_path'], 'br') as f:
+                assert airdrop['file_hash'] == hashlib.sha256(f.read()).hexdigest(), f'Invalid hash for {airdrop["file_path"]}'
+            df = polars.read_parquet(airdrop['file_path'])
+            for (address,) in df.select(polars.selectors.by_index(0)).rows():
+               assert is_checksum_address(address), f'{airdrop["file_path"]} address {address} is not checksummed at row {i+2}'
         else:
+            address = '0xc37b40ABdB939635068d3c5f13E7faF686F03B65'
```

Comment:
I didn't get this one.. is that remaining from some testing?

Reply:
it is because as it was before the address variable was always the last address in the csv file (if you check it comes from iterating the airdrops). Since I removed that the address variable was not asigned. I added one for the tests. Will make it a constant final

Reply:
ah I see, then yeah a final constant or comment will be useful to read